doc/snapshot/building.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Copyright (c) 2020, 2020 Red Hat. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+# Building the snapshot+restore prototype
+
+The snapshot+restore prototype currently builds with the JDK8 Extensions repo and the `snapshot` branch of the eclipse#openj9 and the eclipse#openj9-omr repos.
```

Comment:
It might be worth pointing out that we have only built a snapshot build on x86-64 Linux so far.  The testing that was done on Z for the read-only code cache was a regular build forcing the code generator to produce read-only code.

Reply:
Updated to clarify this